packages/@react-native/tester/overrides.json

```diff
@@ -390,14 +390,14 @@
       "type": "copy",
       "directory": "js/examples/Text",
       "baseDirectory": "packages/rn-tester/js/examples/Text",
-      "baseHash": "528bba6e7f5938ec8c81c54471962883047e68f3",
+      "baseHash": "eafda2ddc809145094056ad5f4ac96146f60061f",
```

Comment:
It looks like changes were made to these files (hence the hash change) but those changes weren't applied to the new files. Did you forget to check in these files?

Reply:
files from rn-tester we've been updating manually however these files are caught by `yarn validate-override` thus linting is failing! Adding the changes to Text should fix this.